lagom-eventstore-example/README.md

```diff
@@ -0,0 +1,137 @@
+# Lagom integration with IBM Project EventStore
+
+[IBM Project EventStore](https://www.ibm.com/us-en/marketplace/project-eventstore) is an in-memory database designed for massive structured data volumes and real-time analytics built on Apache SPARK and Apache Parquet Data Format. 
+
+This project demonstrates a simple Lagom service that includes a [Read-Side](https://www.lagomframework.com/documentation/current/java/ReadSide.html) processor which stores events of the [Persistent Entities](https://www.lagomframework.com/documentation/1.3.x/java/PersistentEntity.html) into Project EventStore for further analysis.
+
+The Lagom service exposes a single HTTP endpoint to simulate events.
+
+## Prerequisites
+
+To build and run this example, you need:
+
+- [git](https://git-scm.com/)
+- [Java SE 8 JDK](http://www.oracle.com/technetwork/java/javase/overview/index.html)
+- [Maven 3.2.1+](https://maven.apache.org/) to build and run the Lagom project (3.5.0 recommended)
+- IBM Project EventStore [downloaded](https://github.com/IBMProjectEventStore/EventStore-DeveloperPreview/releases/tag/1.1.1), [installed](https://www.ibm.com/support/knowledgecenter/SSGNPV/eventstore/desktop/welcome.html) and running on your local machine. (This process requires 12+Gb of downloads)
+
+Once you have an IBM Project EventStore running in your machine, the main steps to run this example are:
+
+1.  [Download and set up the Lagom service](#download-and-set-up-the-lagom-service)
+2.  [Start the Lagom sample application](#start-the-lagom-sample-application)
+3.  [Generate some traffic on the Lagom service](#generate-some-traffic-on-the-lagom-service)
+4.  [Analyse the traffic using IBM Project EventStore Notebooks](#analyse-the-traffic-using-ibm-project-eventstore-notebooks)
+
+## Download and set up the Lagom service
+
+(Make sure you completed all the steps on the Pre-Requisites section and IBM Project EventStore is installed and running)
+
+Follow these steps to get a local copy of this project and configure it to connect it to the IBM Project EventStore running on your machine
+
+1.  Open a command line shell and clone this repository:
+    ```
+    git clone https://github.com/lagom/ibm-integration-examples.git
+    ```
+2.  Change into the root directory for this example:
+    ```
+    cd lagom-eventstore-example
+    ```
+3.  To supply the configuration, do one of the following:
+    1. Open the `lagom-eventstore-impl/src/main/resources/ibm-event-store.conf` file in a text editor and fill in the empty value of the `endpoints` setting.
```

Comment:
It's not totally clear from the instructions what I should be putting here. I tried `localhost:5555` and it seemed to work. Are there circumstances in which it wouldn't work? Can we just make that the default?

Reply:
This is probably an improvement in latest versions of EventStore or Docker on how the binding and announce happens within the infrastructure. This was a pain point originally. I'll test with 127.0.0.1 and if that works we should make that the default, yeah.

Reply:
Hmm, it doesn't seem to fail with flying colors but I think data doesn't make it into EventStore. Either that or the notebook still needs to use a non-localhost ip. Will investigate further.

Reply:
Yes, the notebook does need the public IP for some reason. Probably because it's running inside a Docker container.

Reply:
OK I recall some discussion about this in the past: previously the Event Store Docker container was not mapping a host port by default. I think that was changed in Event Store 1.1.1.

IIRC the notebook is also running in a Docker container, but a separate one than the server runs in, so localhost won't work there. It would be better if it used Docker's networking features to avoid the need to configure manually, but that's out of our hands.

For this project, I think it would be better to default to localhost to reduce the amount of configuration required by most people trying this out.

Reply:
OK I take it back now that I see how you've changed the notebook instructions. Let's keep it as is for consistency.